### PR TITLE
add(case api endpoints)

### PR DESCRIPTION
**Pull request by ChrOertlin**
### Added

Endpoinst to create and get cases to the arnold API

### Fixed

**Steps to consider while deploying**
- Configuration changes:
- Documentation updates:
- Inform users by email:

### Review:
- [ ] Code approved by
- [ ] Tests executed on stage by:  (Document the test done with screen shots and description.)
- [ ] "Merge and deploy" approved by

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: arnold/api/api_v1/endpoints/case.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    response_model=Case,
+)
+def create_case(
+    case: Case = Body(...),
+    adapter: ArnoldAdapter = Depends(get_arnold_adapter),
+) -> JSONResponse:
+    """Create a case document in the database."""
+    if read.case.get_case(case_id=case.id, adapter=adapter):
+        return JSONResponse(
+            status_code=status.HTTP_405_METHOD_NOT_ALLOWED,
```
Comment (seallard) [inline arnold/api/api_v1/endpoints/case.py:25]: 409 might be more descriptive

File: arnold/api/api_v1/endpoints/case.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    "/case/{case_id}",
+    response_description="Get a case document.",
+    status_code=status.HTTP_200_OK,
+    response_model=Case,
+)
+def get_case(
+    case_id: str, adapter: ArnoldAdapter = Depends(get_arnold_adapter)
+) -> Case:
+    """Retrieve a case document from the database."""
+    return read.case.get_case(case_id=case_id, adapter=adapter)
```
Comment (seallard) [inline arnold/api/api_v1/endpoints/case.py:49]: Is it necessary to wrap this in a JSONResponse like the other ones?

File: arnold/api/api_v1/endpoints/case.py (hunk 1 above)
Comment (ChrOertlin) [reply arnold/api/api_v1/endpoints/case.py:25]: I just learned about the HTTP_418_I_AM_A_TEAPOT... I think we should use this everywhere. :)

File: arnold/api/api_v1/endpoints/case.py (hunk 2 above)
Comment (ChrOertlin) [reply arnold/api/api_v1/endpoints/case.py:49]: I actually extended this function a bit more. to now try except with returning an error if it fails.

**Pull request merged**